plugins/ruby-foreman-tasks/debian/control

```diff
@@ -2,7 +2,7 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.7)
+Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.9)
```

Comment:
I think foreman-tasks should be bumped if you change it's spec, I suppose we need 0.6.13-4 in [changelog](https://github.com/theforeman/foreman-packaging/blob/deb/develop/plugins/ruby-foreman-tasks/debian/changelog)